tasks/post_to_catalog/test/unit_tests/test_post_to_catalog.py

```diff
@@ -147,6 +148,7 @@ def test_send_record_to_database_rejects_bad_record(
             "version": uuid.uuid4().__str__(),
         }
         granule = {
+            "providerId": uuid.uuid4().__str__(),
```

Comment:
Depending on what we decide with the input schema, this may need to be removed.

Reply:
removed